### ISSUE TITLE
Props validation error

### ISSUE THREAD

**Issue by mgcrea** [elyalvarado/react-navigation-switch-transitioner#3]
Just tried to integrate this in my app:

```js
const RootNavigator = createSwitchNavigator(
  {
    AuthLoading: AuthLoadingScreen,
    Main: withFadeTransition(MainNavigator),
    Auth: AuthNavigator,
    Register: withFadeTransition(RegisterNavigator)
  },
  {
    initialRouteName: 'AuthLoading'
  }
);
```

And got:

<img width="559" alt="screen shot 2018-09-28 at 02 24 31" src="https://user-images.githubusercontent.com/108273/46181386-b7098380-c2c5-11e8-81d8-2c3b6ab254d6.png">

using `"react-navigation@2.14.0"`

**Comment by elyalvarado** [comment]
Hi @mgcrea Are you using the provided `createSwitchNavigator` or the one from react-navigation? Also how are you doing the props validation?

**Comment by elyalvarado** [comment]
Yes, I was able to reproduce it and will be releasing a fix

**Comment by elyalvarado** [comment]
This should be fixed in v0.0.7, thanks for reporting @mgcrea
